src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -3077,6 +3077,41 @@ because xbuild doesn't support framework reference assemblies.
   </CalculateProjectDependencies>
 </Target>
 
+<PropertyGroup Condition="'$(BuildingInsideVisualStudio)' == 'true'">
+  <EnableCleanAllFiles Condition="'$(EnableCleanAllFiles)' == ''">true</EnableCleanAllFiles>
+  <EnableCleanOnNuGetPackagesChange Condition="'$(CleanOnNuGetPaEnableCleanOnNuGetPackagesChangeckagesChange)' == ''">true</EnableCleanOnNuGetPackagesChange>
```

Comment:
Is `CleanOnNuGetPaEnableCleanOnNuGetPackagesChangeckagesChange` a typo? That looks like gibberish/like someone pasted a word into the middle of an existing string.

Is this intended to be a "public"/user-overridable property? If not, it should start with a `_`. If it *is* public, then it should be documented in [Documentation/guides/BuildProcess.md](https://github.com/xamarin/xamarin-android/blob/master/Documentation/guides/BuildProcess.md).

Reply:
It was a last minute renaming, I will correct it and push the typo fix.

The idea is to have a way of disabling it for the user if for some reason breaks an scenario that we are not aware of. It should not but it's good to have the alternative just in case in my opinion. So it should be public and documented.